mongodb-kubernetes-operator.yaml

```diff
@@ -0,0 +1,78 @@
+#nolint:valid-pipeline-git-checkout-commit,valid-pipeline-git-checkout-tag
+package:
+  name: mongodb-kubernetes-operator
+  version: 0.12.0
+  epoch: 0
+  description: Kubernetes Operator which deploys MongoDB Community into Kubernetes clusters.
+  copyright:
+    - license: MIT
+  dependencies:
+    runtime:
+      - bash
+      - coreutils
+      - dash-binsh
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/mongodb/mongodb-kubernetes-operator
+      tag: v${{package.version}}
+      expected-commit: f3170c62c794768d123b81cf1793525e098891ed
+
+  - uses: go/bump
+    with:
+      deps: |-
+        golang.org/x/net@v0.36.0
+        golang.org/x/oauth2@v0.27.0
+
+  - runs: |
+      mkdir -p ${{targets.contextdir}}/usr/local/bin
+      CGO_ENABLED=0 go build -o manager cmd/manager/main.go
+      cp -av manager ${{targets.contextdir}}/
+      cp -av build/bin/* ${{targets.contextdir}}/usr/local/bin/
```

Comment:
Let's use our `go/build` pipeline here (lots of examples in Wolfi to look at)

We should also never place binaries in `/usr/local/bin` as this path is reserved for users. If we need to provide them there, we should create a compat package